packages/shared/components/inputs/AssetDropdown.svelte

```diff
@@ -1,32 +1,80 @@
 <script lang="typescript">
-    import { FontWeightText } from 'shared/components/Text.svelte'
-    import { Icon, Text } from 'shared/components'
     import { assets } from '@core/wallet'
+    import { AssetTile, Icon, Text } from 'shared/components'
+    import { FontWeightText } from 'shared/components/Text.svelte'
+    import { clickOutside } from 'shared/lib/actions'
 
     export let asset = $assets?.[0]
+
+    const hasMultipleAssets = $assets.length > 1
```

Comment:
should this be following:
`$: hasMultipleAssets = $assets.length > 1` ?

Reply:
good catch, thanks!